js/form.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+  var modal = document.getElementsByClassName("success-modal")[0];
+  var button = document.getElementsByClassName("form-button")[0];
+  var span = document.getElementsByClassName("modal-close")[0]
+
+  span.onclick = function() {
+    modal.classList.add("hidden");
+  }
+
+  window.onclick = function(event) {
+    if (event.target == modal) {
```

Comment:
@mikemorris I agree it's not as clear. This is hiding the modal when clicking on the background, which is part of the modal, and not adding the "hidden" class when clicking on the modal itself. I'll push up this and above changes in a few.